src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1457,5 +1457,10 @@ internal enum ErrorCode
         #endregion more stragglers for C# 7
 
         ERR_BadParameterModifiers = 8205,
+
+        ERR_ByRefConditionalAndAwait = 8301,
+        ERR_RefConditionalNeedsTwoRefs = 8302,
+        ERR_RefConditionalDifferentTypes = 8332,
```

Comment:
ERR_Ref  is much more common, so it will be ERR_RefConditionalAndAwait

---
In reply to: [105267660](https://github.com/dotnet/roslyn/pull/17436#discussion_r105267660) [](ancestors = 105267660)